### PR TITLE
docs: rework the development docs

### PR DESCRIPTION
**Pull request by th33xitus**
* remove outdated Docker page
* rename Standalone page to Environment
* rework instructions to set up a development environment

Signed-off-by: Dominik Willner <th33xitus@gmail.com>

### CODE REVIEW

**Review by meteyou — APPROVED** [review]

File: development/environment.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+{{ page.description }}
+
+## Set up the environment:
+At this point you should have already forked Mainsail into your repositories. \
+If that is not the case, go ahead and fork Mainsail now.
+
+Copy the `.env.development.local.example` file and omit the `.example` at the end. \
+The file is located in the root directory of the Mainsail project.
+
+Now edit the `.env.development.local` file to reflect your printers network configuration. \
```
Comment (lixxbox) [inline development/environment.md:21]: to correct the line break, remove the following blank line

File: development/environment.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+### Install NodeJS
+You can download NodeJS from here: [https://nodejs.org/en/download](https://nodejs.org/en/download) \
+Pick your preferred installation package.
+
+Make sure you run node >=15.9.0
+{: .info}
+
+Open your preferred terminal application and navigate into the Mainsail root directory. \
+Run the follwing command to install all required modules and dependencies:
```
Comment (lixxbox) [inline development/environment.md:54]: `follwing` typo

**Review by lixxbox — CHANGES_REQUESTED** [review]
i have commented on some passages. apart from that, it looks good to me.

**Review by lixxbox — APPROVED** [review]
LGTM 👍

**Pull request merged**